oidc-api/src/test/java/uk/gov/di/authentication/oidc/lambda/AuthorisationHandlerTest.java

```diff
@@ -367,7 +367,7 @@ void shouldRedirectToLoginWhenUserHasNoExistingSessionWithSignedAndEncryptedJwtI
             var captor = ArgumentCaptor.forClass(JWTClaimsSet.class);
             verify(orchestrationAuthorizationService).getSignedAndEncryptedJWT(captor.capture());
             assertEquals(
-                    "{\"userinfo\":{\"email_verified\":null,\"verified_mfa_method_type\":null,\"email\":null}}",
+                    "{\"userinfo\":{\"email_verified\":null,\"current_credential_strength\":null,\"verified_mfa_method_type\":null,\"email\":null}}",
```

Comment:
Not your code, but these tests are quite flaky. I must've missed updating this one when I made the change. Can you replace this with something like the others have, where we compare JSON objects rather than strings?